src/corecel/math/Atomics.hh

```diff
@@ -24,20 +24,17 @@ CELER_FORCEINLINE_FUNCTION T atomic_add(T* address, T value)
 {
 #if CELER_DEVICE_COMPILE
     return atomicAdd(address, value);
-#elif defined(_OPENMP)
+#else
     CELER_EXPECT(address);
     T initial;
-#    pragma omp atomic capture
+#    if defined(_OPENMP)
+#        pragma omp atomic capture
+#    endif
```

Comment:
@paulromano Can you confirm this should work for OpenMP? (I.e., the preprocessor macros around the pragma shouldn't break it, right?)